c2corg_ui/static/js/apiservice.js

```diff
@@ -88,7 +88,7 @@ app.Api.prototype.putJson_ = function(url, json) {
 /**
  * @param {string} url Url suffix
  * @return {!angular.$http.HttpPromise}
- * @private
+ * @export
```

Comment:
then how do you want me to access it without passing through `readDocument` because it's not what I want? shall I create another readDocument2(URL) that calls getJson(url) ? :P